dotcom-rendering/src/components/Badge.tsx

```diff
@@ -33,12 +44,13 @@ const badgeLink = css`
 type Props = {
 	imageSrc: string;
 	href: string;
+	isInLabsSection?: boolean;
 };
 
-export const Badge = ({ imageSrc, href }: Props) => {
+export const Badge = ({ imageSrc, href, isInLabsSection = false }: Props) => {
 	return (
 		<a href={href} css={badgeLink} role="button">
-			<img css={imageStyles} src={imageSrc} alt="" />
+			<img css={imageStyles(isInLabsSection)} src={imageSrc} alt="" />
```

Comment:
Could we use an [array of styles](https://emotion.sh/docs/@emotion/css#array-of-object-styles) here instead?

```suggestion
			<img css={[imageStyles, isInLabsSection ? labsSectionBadgeSizingStyles : frontsSectionBadgeSizingStyles]} src={imageSrc} alt="" />
```